sort/selectionSort.py

```diff
@@ -1,29 +1,29 @@
 # O(n^2)
 
-def selectionSort (List):
+def selection_sort (l):
   '''
     Sorts a list in increasing order.
     First, selects the smallest number and swaps it with the first number.
     Then selects the second smallest number and swaps it with the second number.
     And so on.
   '''
-  for i in range( len(List) - 1 ):
-    nextSmallest_i = getNextSmallest(List, i)
-    (List[i], List[nextSmallest_i]) = (List[nextSmallest_i], List[i])
+  for i in range( len(l) - 1 ):
+    next_smallest_i = get_next_smallest(l, i)
+    (List[i], List[next_smallest_i]) = (List[next_smallest_i], List[i])
     
     
-def getNextSmallest (List, startIndex):
+def get_next_smallest (l, start_index):
   '''
     Helper function to find the index of the smallest number
     in the portion of the list after the start index.
     Returns the index of the number.
   '''
-  minValue = List[startIndex]; 
-  index = startIndex;
+  min_value = l[start_index]; 
```

Comment:
can you use another name for the 'l' variable? It does really look like 1 and I.